common/src/main/java/tds/common/configuration/EventLoggerConfiguration.java

```diff
@@ -1,9 +1,36 @@
 package tds.common.configuration;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.handler.MappedInterceptor;
+
+import tds.common.web.interceptors.EventLoggerInterceptor;
 
 @Configuration
 @ComponentScan(basePackages = "tds.common.logging")
 public class EventLoggerConfiguration {
+  private final ApplicationContext applicationContext;
+  private final ObjectMapper objectMapper;
+
+  @Autowired
+  public EventLoggerConfiguration(final ApplicationContext applicationContext,
+                                  final ObjectMapper objectMapper) {
+    this.applicationContext = applicationContext;
+    this.objectMapper = objectMapper;
+  }
+
+  /**
+   * Log incoming http requests formatted for logstash centralized logging
+   *
+   * @return http interceptor that delegates calls to {@link EventLoggerInterceptor}
+   */
+  @Bean
+  public MappedInterceptor eventInterceptor()
```

Comment:
When using the `@Bean` annotation in a configuration class I usually pass in the dependencies that are needed rather than wire them into the constructor.  Spring will pass in the dependencies when things have the `@Bean` annotation and throw if something is required and cannot be found in the Spring container.  The benefit to using that style rather than the constructor style used above is we could configure 10 beans in this class with a bunch of different dependencies and the constructor wouldn't grow.  If this wasn't a `@Configuration` annotated class I'd prefer what you have done.

So this would be:
```
@Bean
public MappedInterceptor eventInterceptor(final ApplicationContext applicationContext, final ObjectMapper objectMapper) {
  return new MappedInterceptor(null, new EventLoggerInterceptor(applicationContext.getId(), objectMapper));
}
```